src/main/java/com/yello/server/domain/friend/dto/response/RecommendFriendResponse.java

```diff
@@ -0,0 +1,28 @@
+package com.yello.server.domain.friend.dto.response;
+
+import com.yello.server.domain.user.entity.User;
+import lombok.Builder;
+
+public record RecommendFriendResponse(
+        Long id,
+        String name,
+        String schoolData,
+        String profileImage
+) {
+    public static RecommendFriendResponse of(User user) {
+        return RecommendFriendResponse.builder()
+                .id(user.getId())
+                .name(user.getName())
+                .schoolData(user.getGroup().getSchoolName() +  " " + user.getGroup().getDepartmentName() + " " +user.getGroup().getAdmissionYear()+ "학번")
```

Comment:
필드명은 schoolData -> group 어떠신가요?? 
추후에 학교 이외의 그룹이 더 생길 수도 있을 것 같아서요!

그리고, 그룹을 하나의 문자열로 만드는건 School 엔티티에 toString()을 오버라이딩 해둔 것이 있어서
고거 활용하면 될 것 같습니다.

```suggestion
public record RecommendFriendResponse(
        Long id,
        String name,
        String group,
        String profileImage
) {
    public static RecommendFriendResponse of(User user) {
        return RecommendFriendResponse.builder()
                .id(user.getId())
                .name(user.getName())
                .group(user.getGroup().toString())
                ...
```

Reply:
오 수정하겠습니다!